ApiDemos/kotlin/app/src/gms/java/com/example/kotlindemos/StreetViewPanoramaEventsDemoActivity.kt

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.example.kotlindemos
+
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.gms.maps.StreetViewPanorama
+import com.google.android.gms.maps.StreetViewPanorama.*
+import com.google.android.gms.maps.SupportStreetViewPanoramaFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.StreetViewPanoramaCamera
+import com.google.android.gms.maps.model.StreetViewPanoramaLocation
+import com.google.android.gms.maps.model.StreetViewPanoramaOrientation
+
+/**
+ * This shows how to listen to some [StreetViewPanorama] events.
+ */
+class StreetViewPanoramaEventsDemoActivity : AppCompatActivity(),
+    OnStreetViewPanoramaChangeListener, OnStreetViewPanoramaCameraChangeListener,
+    OnStreetViewPanoramaClickListener, OnStreetViewPanoramaLongClickListener {
+
+    private var streetViewPanorama: StreetViewPanorama? = null
+    private var panoChangeTimesTextView: TextView? = null
+    private var panoCameraChangeTextView: TextView? = null
+    private var panoClickTextView: TextView? = null
+    private var panoLongClickTextView: TextView? = null
```

Comment:
All the `View` members can be declared with `lateinit` so that they are non-optional. e.g. `private var lateinit streetViewPanorama: StreetViewPanorama`, etc.

Reply:
Sounds good! I just made the change in https://github.com/googlemaps/android-samples/pull/207/commits/3234f9836ed8661b5a4461b4a4f38346f7417433.